DependencyInjection/CmfSeoExtension.php

```diff
@@ -233,9 +233,18 @@ private function loadSitemapHandling($config, XmlFileLoader $loader, ContainerBu
     {
         $loader->load('sitemap.xml');
 
-        $container->setParameter(
-            $this->getAlias().'.sitemap.default_change_frequency',
-            $config['default_chan_frequency']
-        );
+        $container->setParameter($this->getAlias().'.sitemap.configurations', $config['configurations']);
+
+        foreach ($config['configurations'] as $key => $configuration) {
+            $container->setParameter($this->getAlias().'.sitemap.'.$key.'_configuration', $configuration);
+        }
+
+        // set a base default change frequency by now todo change that when votes respect sitemap by name
+        if (isset($config['configurations']['default']['default_change_frequency'])) {
```

Comment:
should we autocreate the default sitemap if its not explicitly defined? 

and how do we handle default templates? we really need #217 now but i would say: if no templates are defined, we add the default ones for html and xml, otherwise we don't do anything extra and assume the user did what he wanted.